src/javascript/app_2/Stores/Modules/Portfolio/portfolio-store.js

```diff
@@ -20,6 +20,7 @@ export default class PortfolioStore extends BaseStore {
     @observable positions  = [];
     @observable is_loading = false;
     @observable error      = '';
+    getPositionById = (id) => this.positions.find((position) => +position.id === +id);
```

Comment:
nitpick, `(id)` -> `id`

Reply:
I agree with spacing but disagree with the removal of `parenthesis`.